core/vm/interpreter.go

```diff
@@ -225,7 +222,7 @@ func (in *EVMInterpreter) Run(contract *Contract, input []byte, readOnly bool) (
 		// Get the operation from the jump table and validate the stack to ensure there are
 		// enough stack items available to perform the operation.
 		op = contract.GetOp(pc)
-		operation := in.cfg.JumpTable[op]
+		operation := &in.jt[op]
```

Comment:
If there is any performance gain, it's likely from this change instead of the change in NewEVMInterpreter because this statement executes every intstruction.